src/Raven.Server/Documents/Indexes/Persistence/Lucene/LuceneIndexWriteOperation.cs

```diff
@@ -192,7 +192,7 @@ public override (long RamSizeInBytes, long FilesAllocationsInBytes) GetAllocatio
                 }
             }
 
-            return (usedMemory, fileAllocations);
+            return (RamSizeInBytes: usedMemory, FilesAllocationsInBytes: fileAllocations, UnmanagedAllocationsInBytes: 0);
```

Comment:
@grisha-kotler do we have any native allocations in Lucene that we can return here? So that would be specific to the index writer. 

I think the changes you did in Lucene to use native memory were applied to caching and querying but wanted to check with you.